nuxt.config.js

```diff
@@ -66,6 +66,7 @@ export default {
         "~/plugins/craft.js",
         "~/plugins/data-api.js",
         "~/plugins/elasticsearchplugin.js",
+        { src: '~/plugins/get-headers.js', mode: 'client' },
```

Comment:
rename the file to be .client.js, and remove mode, this is pattern we are following